Index: src/main/java/rofla/back/back/model/SubjectInfo.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package rofla.back.back.model;\n\nimport jakarta.persistence.*;\nimport lombok.Getter;\nimport lombok.Setter;\n\n@Getter\n@Setter\n@Entity\n@Table(name = \"subject_info\", schema = \"grupgo\")\npublic class SubjectInfo {\n    @Id\n    @GeneratedValue(strategy = GenerationType.IDENTITY)\n    @Column(name = \"id\", nullable = false)\n    private Long id;\n\n    @Column(name = \"name\", nullable = false, length = 30)\n    private String name;\n\n    @Column(name = \"subject_num\", nullable = false, length = 30)\n    private String subjectNum;\n\n    @Column(name = \"start_time\")\n    private String startTime;\n\n    @Column(name = \"end_time\")\n    private String endTime;\n\n    @Column(name = \"classroom\", length = 30)\n    private String classroom;\n\n    @Column(name = \"professor\", nullable = false, length = 30)\n    private String professor;\n\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/rofla/back/back/model/SubjectInfo.java b/src/main/java/rofla/back/back/model/SubjectInfo.java
--- a/src/main/java/rofla/back/back/model/SubjectInfo.java	(revision df9ae623847db1970568377fa15af837fcca5340)
+++ b/src/main/java/rofla/back/back/model/SubjectInfo.java	(date 1721575902268)
@@ -26,7 +26,7 @@
     @Column(name = "end_time")
     private String endTime;
 
-    @Column(name = "classroom", length = 30)
+    @Column(name = "classroom", length = 300)
     private String classroom;
 
     @Column(name = "professor", nullable = false, length = 30)
